### DISCUSSION TITLE
Welcome to colpat Discussions!

### DISCUSSION THREAD

**Discussion by VishwaGauravIn** [VishwaGauravIn/colpat#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Not-Qualified** [comment]
It's very good that what you've achieved

I'm from India, Are you a designer ?

Do you design things ?

You sure are a programmer, that's for sure

**Reply by VishwaGauravIn** [reply]
Yes I have designed as well as coded this.

**Reply by Not-Qualified** [reply]
> Yes I have designed as well as coded this.

So, You are good at designing things that's for sure

Can you make some custom emoji and GIFs ?

**Reply by VishwaGauravIn** [reply]
Sorry, I don't do that stuff. Only web design

**Reply by Not-Qualified** [reply]
> Sorry, I don't do that stuff. Only web design

Got it, no worries, I've some things that I want create and it's something related to emojis

**Reply by VishwaGauravIn** [reply]
Oh, okay
